addon/components/layers/wms-single-tile-layer.js

```diff
@@ -11,6 +12,74 @@ import WmsLayerComponent from './wms-layer';
   @extends WMSLayerComponent
  */
 export default WmsLayerComponent.extend({
+  /**
+    Performs 'getFeatureInfo' request to WMS-service related to layer.
+
+    @param {<a href="http://leafletjs.com/reference-1.0.0.html#latlng">L.LatLng</a>} latlng Identification point coordinates.
+  */
+  _getFeatureInfo(latlng) {
+    let layer = this.get('_leafletObject');
```

Comment:
it is possibly better to make wms-single-tile layer like wms-wfs one.
create inner wms layer and use it for featureInfo methods.